ftw/inflator/__init__.py

```diff
@@ -6,3 +7,13 @@
 
 
 IS_PLONE_5 = get_distribution('Plone').version >= '5'
+
+HAS_MULTILINGUAL = False
+IS_PLONE_APP_MULTILINGUAL_2 = False
+
+try:
+    dist = get_distribution('plone.app.multilingual')
+    IS_PLONE_APP_MULTILINGUAL_2 = dist.version >= '2'
```

Comment:
Should version 3.x result in `IS_PLONE_APP_MULTILINGUAL_2 == True` ?

Reply:
👍 updated, I now check for plone5 and is_plone_app_multilingual_2